src/client/state.rs

```diff
@@ -474,7 +474,8 @@ mod tests {
                            assert_eq!(broker.map(|b| b.1), state.find_broker(topic, id));
                            (id, broker)
                        })
-                       .collect::<Vec<_>>()[..]);
+                       .collect::<Vec<_>>()
+                        [..]);
     }
```

Comment:
`cargo fmt` seems to suggest the original was just fine. maybe just unintended newline here?